ansible/master/ansible-jenkins/tasks/config.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Configure Jenkins Port
+  lineinfile: dest=/etc/sysconfig/jenkins regexp=^HTTP_PORT= line=HTTP_PORT={{port}}
```

Comment:
formatting nitpick.

Also, I believe you need 'backrefs: yes' here to make sure it won't keep adding this line over and over again on each subsequent run.